styles/_global.scss

```diff
@@ -3,7 +3,4 @@
   padding: 0;
   margin: 0;
 }
```

Comment:
Do we need that considering we have Normalize.css?

Reply:
Yes, normalize doesn't do that. It gives us more control over the design